docs/tutorials/td-tutorial.md

```diff
@@ -0,0 +1,224 @@
+# Send data securely to Arm Treasure Data
+
+There are two ways to send data securely from Mbed OS to Treasure Data:
+
+- HTTPS library - Send data directly to the Treasure Data REST API.
+- Fluentd using fluent logger library - Send data to a hosted Fluentd instance that aggregates and forwards the data on to your treasure data account.
```

Comment:
What are the rules for capitalizing "fluent," "fluentd" and "fluend"?

Reply:
not super sure but i went to https://www.fluentd.org/ and it looked like it was init capped in text, lower case in logos.....kinda like Arm.